src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -0,0 +1,187 @@
+service needs to have knowledge and connect to a SMTP server to use it.
+Relevant data is specified in the properties file located
+* `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/email.properties`
+
+By default Transport Layer Security is active, change to false to deactivate it:
+----
+mail.smtp.starttls.enable=true
+----
+So is the authentication to the STMP server.
+----
+mail.smtp.auth=true
+----
+
+Provide the server port used to connect
+----
+mail.smtp.port=
+----
+
+The credentials used to connect
+----
+mail.smtp.username=
+mail.smtp.password=
+----
+
+The host of the SMTP server
+----
+mail.smtp.host=
+----
+
+The email used to broadcast the emails is defined in the application.properties file,
+go to <<Housekeeping>> to see where it is located.
+It is the email that will appear in the "From" field of the sent emails
+----
+notification.email.from=
+----
+
+[[_notification_third_party]]
+===== Third party
+
+====== Configuration
+
+The Third party notification method executes a script when an event occurs.
+Location of the script is specified either via a path, relative to the Scheduler or absolute.
+
+----
+notification.script.path=dist/war/notification-service/WEB-INF/classes/exampleScript.sh
+----
+
+Or a URL
+
+----
+notification.script.url=
+----
+
+Note that to the URL will be processed if the path is empty
+
+An optional parameter can be provided to specify the engine to be used to execute the script
+if not provided the extension of the file will determine the appropriate engine to use
+
+----
+#optional parameter for specifying Script Engine
+notification.script.engine=bash
+----
+
+Finally multiple script execution can be enable to execute the script for each user
+that has an active subscription for an event and has Third party notification method activated.
+It is set to false by default.
+
+For example if 5 users are subscribed to the Scheduler Frozen event with the Third party notification
+method active and multiple execution enabled, then each time the  <<_glossary_proactive_scheduler,*ProActive Scheduler*>> freezes
+the script will be executed one time per notified user so in this example 5 times.
+If it is set to false only one execution will be performed regardless of how many users are notified.
+
+----
+#if false execute the script one time per event even if there are several subscribers
+#if true execute the script for each subscribers for each event
+notification.script.multiple.execution=false
+----
+
+====== Reading notification data
+
+Notification data is passed as bindings to the script, therefore you can access it simply by reading
+the corresponding field.
+In some cases some fields might be null, for instance there is no BUCKET_NAME, PROJECT_NAME, WORKFLOW_NAME
+for Scheduler events.
+
+Example in groovy:
+----
+println "Reading the notification message: " + MESSAGE
+----
+
+In the following table is specified if fields have a value regarding the notification type
```

Comment:
The following table depicts which fields can be set according to notification types.  (do you agree?)

Reply:
I will clarify this part.
It is not about setting fields but reading the fields automatically passed to the script as bindings.
Those fields hold notification related data such as creation date, event type ...
The goal here is to specify if available fields have a value depending on the event type, like in my example where a scheduler event is not binded to a job thus the JOB_ID field will be null.